src/tools/tools/LoggerCSharp.fs

```diff
@@ -47,6 +47,18 @@ type Logger private() =
         if Logger.LoggerProvider.IsEnabled(logId, logLevel) then
             Logger.LoggerProvider.Log((logId, logLevel, messageFunc.Invoke()))
 
+    /// Log message generated by "messageFunc" using "JobId", 
+    /// "messageFunc" is evaluated only if "logLevel" is less than or equal to the log level for "logId"
+    static member inline LogF(jobID : Guid, logLevel : LogLevel,messageFunc : Func<string>) =
```

Comment:
This is to be used by C#, probably no need to "inline" to be consistent with other APIs.